crates/krator-derive/Cargo.toml

```diff
@@ -16,20 +16,37 @@ license-file = "../../LICENSE"
 description = "Derive macros for the Krator crate"
 repository = "https://github.com/deislabs/krustlet"
 readme = "README.md"
-keywords = [
-    "wasm",
-    "wasi",
-    "webassembly",
-    "kubernetes",
-]
+keywords = ["wasm", "wasi", "webassembly", "kubernetes"]
 
 [lib]
 proc-macro = true
-
 [dependencies]
 syn = "1.0"
 quote = "1.0"
 
+anyhow = { version = "1.0.40", optional = true }
+k8s-openapi = { version = "0.11", default-features = false, features = ["v1_20"], optional = true }
+kube = { version = "0.52", default-features = false, features = ["derive"], optional = true }
+proc-macro2 = "1.0.24"
+rcgen = { version = "0.8.9", features = ["x509-parser", "pem"], optional = true }
+schemars = { version = "0.8.0", optional = true }
+serde_json = { version = "1.0", optional = true }
+serde_yaml = { version = "0.8", optional = true }
+
+[features]
+default = ["kube/native-tls"]
+admission-webhook = ["kube", "k8s-openapi", "rcgen", "anyhow"]
+
 [package.metadata.docs.rs]
 features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
+
+[dev-dependencies]
+kube = { version = "0.52", default-features = false, features = ["derive"] }
+kube-runtime = { version = "0.52", default-features = false }
+kube-derive = "0.52"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+serde_yaml = "0.8"
+schemars = "0.8.0"
+krator = { default-features = false, path = "../krator" }
```

Comment:
This creates a circular dependency when trying to push the crate. We had this in here before (I think)

Reply:
good catch ... although I think it's fine as it is only in `dev-dependencies`.
I removed it as it is not needed anymore (it was part of a test before)

removed in commit c3391b9